zapisy/apps/enrollment/courses/views.py

```diff
@@ -76,9 +77,14 @@ def course_view_data(request, slug) -> Tuple[Optional[CourseInstance], Optional[
         student = request.user.student
 
     groups = course.groups.select_related(
-        'teacher',
-        'teacher__user',
-    ).prefetch_related('term', 'term__classrooms', 'guaranteed_spots', 'guaranteed_spots__role')
+        'teacher', 'teacher__user',
+    ).prefetch_related(
+        'term', 'term__classrooms', 'guaranteed_spots', 'guaranteed_spots__role'
+    ).annotate(
```

Comment:
```suggestion
    ).alias(
```

Jeżeli nie korzystamy nigdzie dalej z pól pochodzących z adnotacji, to być może metoda `alias` ([https://docs.djangoproject.com/en/4.2/ref/models/querysets/#alias](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#alias)) jest bardziej adekwatna?

Reply:
Huh, opis brzmi tak, jak dokładnie skrojony pod to, co chcemy zrobić. Skomituję tę sugestię żeby przetestować, najwyżej się ją cofnie…

Reply:
… kiedyś może to zrobimy, jak już wreszcie zaktualizuję Django do wersji 3.2 :clown_face: